src/component/handlers/edit/DraftEditorEditAndroidHandler.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DraftEditorEditAndroidHandler
+ * @flow
+ */
+
+'use strict';
+
+const onBeforeInput = require('editOnBeforeInputAndroid');
+const onCompositionStart = require('editOnCompositionStart');
+const onSelect = require('editOnSelect');
+
+const noop = () => {};
+
+// Noop everything except beforeInput and select...
+const DraftEditorEditAndroidHandler = {
+  onBeforeInput,
+  onCompositionStart,
+  onSelect,
+  onBlur: noop,
```

Comment:
< 1 ⭐️ : It should be safe to omit these instead of having to pass a noop.

Reply:
sounds good, i will verify + remove